### PR TITLE
Fix typo.

### PR DESCRIPTION
**Pull request by cranst0n**

### CODE REVIEW

File: website/layouts/index.ace [hunk 1]
```diff
@@ -30,7 +30,7 @@ html lang={{.Site.LanguageCode}}
                 .col-sm-6
                   .panel.panel-primary
                     .panel-heading Proven at Twitter Scale
-                    .panel-body Heron powers all realtime analytics with varied use cases at Twitter for over 2 years. Incident reports dropped by an order of magnitude demonstrating proven reliability and scalability
+                    .panel-body Heron has powered all realtime analytics with varied use cases at Twitter for over 2 years. Incident reports dropped by an order of magnitude demonstrating proven reliability and scalability
```
Comment (billonahill) [inline website/layouts/index.ace]: Good catch. If I could request one more edit please, we should replace "for over 2 years" with "since 2014".

**Comment by billonahill** [comment]
Thanks @cranst0n, looks good! Would you please accept the CLA as described here:
http://twitter.github.io/heron/docs/contributors/community/

**Comment by cranst0n** [comment]
@billonahill Signed!

**Comment by billonahill** [comment]
Thanks!

**Pull request merged**